### PR TITLE
feat: generic analytics for verification events

### PR DESCRIPTION
**Pull request by HJunyuan**
## What does this PR do?

- [x] Implement Google Analytics events for all OA v2 documents that have been successfully/unsuccessfully verified
- [x] Add new fields into Custom Dimensions on Google Analytics (dev & prod): `issuer_identity_location`, `template_name` and `template_url`

### CODE REVIEW

File: utils/google-analytics.ts [hunk 1]
```diff
@@ -50,32 +56,40 @@ export const useGoogleAnalytics = (): void => {
...
 
-export const sendHealthCertErrorEvent = (data: OpenAttestationDocument, fragments: VerificationFragment[]): void => {
-  if (!isHealthCert(data)) {
-    return;
-  }
+export const sendUnsuccessfulVerificationEvent = (
+  data: OpenAttestationDocument,
+  fragments: VerificationFragment[]
+): void => {
+  if (utils.isRawV3Document(data)) return; // OA v3 currently not supported by verify.gov.sg
```
Comment (zixiang2018) [inline utils/google-analytics.ts:79]: Do we wanna log this as an error event? Or we just gonna skip this until it becomes supported, and then we will remove this line?

Comment (HJunyuan) [reply utils/google-analytics.ts:79]: Good point, I think we can log it. At the same time, I don't think we have any v3 documents generated for `verify.gov.sg` also 🤔

Comment (zixiang2018) [reply utils/google-analytics.ts:79]: We can update this when verify.gov.sg needs to be compatible with v3 documents in the future 👍

**Review by zixiang2018 — APPROVED** [review]
🚀

**Pull request merged**